src/main/java/com/aws/greengrass/mqttbridge/MQTTBridge.java

```diff
@@ -179,6 +167,61 @@ public void shutdown() {
         }
     }
 
+    public class CertificateAuthorityChangeHandler {
+
+        private BatchedSubscriber subscriber;
+
+        @SuppressWarnings({"unchecked", "PMD.UnusedFormalParameter"})
+        private void onCAChange(WhatHappened what, List<Node> whatChanged) {
+            if (whatChanged.isEmpty()) {
+                return;
+            }
+
+            List<String> caCerts = (List<String>) whatChanged.get(whatChanged.size() - 1).toPOJO();
+            if (Utils.isEmpty(caCerts)) {
+                return;
+            }
```

Comment:
this scares me. Perform a node lookup at this point instead of reading from what node list. Then use `Coerce.toStringList`

Reply:
yeah, I think I was trying to be too slick by avoiding a lookup, will switch to a lookup instead 👍

Reply:
fixed